cumulusci/tasks/bulkdata/generate_mapping.py

```diff
@@ -161,36 +168,32 @@ def _build_mapping(self):
         objs = set(self.schema.keys())
         stack = self._split_dependencies(objs, self.refs)
 
-        field_sort = (
-            lambda f: "  " + f
-            if f == "Id"
-            else (" " + f if f in self.core_fields else f)
-        )
+        def strip_namespace(element):
+            ns = self.project_config.project__package__namespace
+            if self.options["strip_namespace"] and ns and element.startswith(ns):
```

Comment:
We should be checking if it starts with ns + "__" since there could be multiple namespaces in the org where one is a substring of another.

Reply:
Also while there's no particular need to micro-optimize this, the assignment of `ns` could be moved outside the closure.